src/components/blog/ReactionButton.css

```
@@ -1,246 +1,372 @@
 
 .rb-icon {
-  font-size: 15px;
-  line-height: 1;
-  transition: transform 0.18s cubic-bezier(0.34, 1.56, 0.64, 1);
+  width: 24px;
+  height: 24px;
+  border-radius: 999px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  color: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)